lib/external/math.dart

```diff
@@ -40,4 +50,128 @@ ASTNode funcRand(Runtime runtime, ASTNode self, List<ASTNode> args) {
   }
 
   throw const UnexpectedTypeException('The rand method only takes [String, double, int, and bool] argument types');
+}
+
+ASTNode funcSin(Runtime runtime, ASTNode self, List<ASTNode> args){
+  runtimeExpectArgs(args, [ASTType.AST_ANY]);
```

Comment:
Should this be AST_INT as the arg? Or sin can be done on a string also?

Reply:
It could be done on both double and int. Anything else should throw an error

Reply:
Ah sorry for that!

Reply:
I believe it has been resolved then

Reply:
Nope